static/gsApp/views/subscriptionPage/usageTotals.tsx

```diff
@@ -184,6 +184,10 @@ export function calculateCategoryPrepaidUsage(
   prepaidSpend: number;
   prepaidUsage: number;
 } {
+  // @ts-expect-error TS(7053): Element implicitly has an 'any' type because expre... Remove this comment to see the full error message
+  const categoryInfo: BillingMetricHistory = subscription.categories[category];
```

Comment:
i think we can resolve the ts error
```suggestion
  const categoryInfo: BillingMetricHistory | null = subscription.categories[category];
```
or maybe something like
```suggestion
  const categoryInfo: BillingMetricHistory = subscription.categories[category] as BillingMetricHistory;
```

Reply:
Also remove `// @ts-expect-error TS(7053)` comment and try to resolve the types.